timescaledb/how-to-guides/backup-and-restore/docker-and-wale.md

```diff
@@ -1,206 +1,194 @@
+      -carchive_timeout=600 \
+      -ccheckpoint_timeout=700 \
+      -cmax_wal_senders=1
+    ```
+1.  Run TimescaleDB within Docker:
+    ```bash
+    docker exec -it timescaledb psql -U postgres
+    ```
+
+## Run the WAL-E sidecar
+The [WAL-E Docker image][wale image] runs a web endpoint that accepts WAL-E
+commands across an HTTP API. This allows PostgreSQL to communicate with the
+WAL-E sidecar over the internal network to trigger archiving. You can also use
+the container to invoke WAL-E directly. The Docker image accepts standard WAL-E
+environment variables to configure the archiving backend, so you can issue
+commands from services such as AWS S3. See [WAL-E's documentation][wale
 official] for more details.
 
-To enable the WAL-E docker image to perform archiving, it needs to use
-the network and data volume(s) of the TimescaleDB container. It also
-needs to know the location of the write-ahead log and data
-directories. Thus, launch the WAL-E sidecar as follows:
-
-```bash
-docker run​ \
-    --name wale \
-    --network timescaledb-net \
-    --volumes-from timescaledb \
-    -v ~/backups:/backups \
-    -e WALE_LOG_DESTINATION=stderr \
-    -e PGWAL=/var/lib/postgresql/data/pg_wal \
-    -e PGDATA=/var/lib/postgresql/data/pg_data \
-    -e PGHOST=timescaledb \
-    -e PGPASSWORD=insecure \
-    -e PGUSER=postgres \
-    -e WALE_FILE_PREFIX=file://localhost/backups \
-    timescale/timescaledb-wale:latest
-```
-
-This will make the WAL-E image listen on commands on port 80 on the
-`timescaledb-net` internal network and write backups to `~/backups` on
-the Docker host.
-
-To do the initial base backup, execute the following command in the
-running WAL-E container (assuming the container's name is
-`timescaledb-wale`):
-
-```bash
-docker exec wale wal-e backup-push /var/lib/postgresql/data/pg_data
-```
-
-Alternatively, do it via the sidecar's HTTP endpoint (this requires
-exposing the sidecar's port `80` on the Docker host by mapping it to,
-e.g., port `8080`):
-
-```bash
-curl http://localhost:8080/backup-push
-```
-
-Base backups should be done at regular intervals (e.g., every day) to
-minimize the amount of WAL replay, making recoveries faster. To make
-new base backups, simply re-trigger a base backup as shown above,
-either manually or on a schedule (e.g., via a CRON job). If you run
-TimescaleDB on Kubernetes, there is built-in support for scheduling
-cron jobs that can invoke base backups via, e.g., the WAL-E
-container's HTTP API.
+To enable the WAL-E docker image to perform archiving, it needs to use the same
+network and data volumes as the TimescaleDB container. It also needs to know the
+location of the write-ahead log and data directories. You can pass all this
+information to WAL-E when you start it. In this example, the WAL-E image listens
+for commands on the `timescaledb-net` internal network at port 80, and writes
+backups to `~/backups` on the Docker host.
+
+### Running the WAL-E sidecar
```

Comment:
I realize this isn't changing much content (if any) below, but reading over this it's not clear to me that this section will actually perform the backup. WAL-E is certainly an advanced topic so maybe we expect more experience (and if so should state somewhere), but if I'm coming to these pages looking for backup how-tos, I'd expect a "Backup" section to complement the "Recovery" section. It's just not clear to me that "running the wal-e sidecar" is going to make my backup.

Does that make sense?

Reply:
Yep, agreed. Updated to "Perform/ing the backup using the WAL-E sidecar".